elementary/utils/time.py

```diff
@@ -70,6 +72,16 @@ def convert_datetime_utc_str_to_timezone_str(
         return isoformat_datetime
 
 
+def convert_datetime_utc_str_to_timezone_datatime(
```

Comment:
We already have the following 2 methods:
`convert_utc_iso_format_to_datetime`
`convert_utc_time_to_timezone`
You can use both or at least call those 2 inside this method instead of rewriting the logic that might be change in one place but be left the same on the other 🙂

Reply:
If we do keep this method that calls the other methods, can you add tests for it please 🙂